modules/abseil-cpp/metadata.json

```diff
@@ -1,4 +1,5 @@
 {
+    "$schema": "../../metadata.schema.json",
```

Comment:
Do we expect different `metadata.json` file to have different schema? If not, can we extract this out and avoid changing every existing metadata.json files? Otherwise, the publish-to-bcr app may also need corresponding changes?

Reply:
I can remove it from this PR, and will send a separate one explaining why this is useful.